solr/solrj/src/java/org/apache/solr/client/solrj/request/beans/PluginMeta.java

```diff
@@ -48,6 +48,7 @@ public PluginMeta copy() {
     result.name = name;
     result.klass = klass;
     result.version = version;
+    result.pathPrefix = pathPrefix;
```

Comment:
Looks like it was forgotten.

Reply:
It was making my test to fail, because `ContainerPluginsRegistry#getPlugin` calls `copy` and `pathPrefix` was lost.